src/components/annotations/draw.js

```diff
@@ -106,12 +116,33 @@ function drawOne(gd, index) {
             });
         });
 
-    // another group for text+background so that they can rotate together
-    var annTextGroup = annGroup.append('g')
-        .classed('annotation-text-g', true)
-        .attr('data-index', String(index));
-
-    var annTextGroupInner = annTextGroup.append('g');
+    if(options.hovertext) {
+        annTextGroupInner
+        .on('mouseover', function() {
+            var hoverOptions = options.hoverlabel;
+            var hoverFont = hoverOptions.font;
+            var bBox = this.getBoundingClientRect();
+            var bBoxRef = gd.getBoundingClientRect();
+
+            Fx.loneHover({
```

Comment:
added a new attribute `captureevents` in https://github.com/plotly/plotly.js/pull/1573/commits/da1e9d70be9f51656e7ce6c67703fe8ae371e6fc

This one defaults to false (so data point hover comes through) unless `hovertext` is provided. But as noted in the description, if you use `plotly_clickannotation` events you must explicitly turn this on.

@cldougl once this is merged we should update the (currently broken) example in https://plot.ly/javascript/text-and-annotations/#styling-and-formatting-annotations, and add a section about `plotly_clickannotation` in https://plot.ly/javascript/plotlyjs-events/ that specifically mentions `captureevents`.